spec/e2e/browser_spec.rb

```diff
@@ -58,7 +58,7 @@ def browser
   it 'redirects /pun/sys/file-editor and preserves the file' do
     browser.goto "#{ctr_base_url}/pun/sys/file-editor/edit/home/ood/.bashrc"
     expect(browser.url).to eq("#{ctr_base_url}/pun/sys/dashboard/files/edit/fs/home/ood/.bashrc")
-    expect(browser.title).to eq('File Editor - Open OnDemand - .bashrc')
+    expect(browser.title).to eq('Dashboard - Open OnDemand')
```

Comment:
No we shouldn't change the test, we should change the application. Page titles are actually very important for blind users to indicate what that page is all about as they go through their open tabs.